azure-container-apps-maven-plugin/src/test/java/com/microsoft/azure/maven/containerapps/configuration/AppContainerMavenConfigTest.java

```diff
@@ -0,0 +1,71 @@
+package com.microsoft.azure.maven.containerapps.configuration;
+
+import com.microsoft.azure.maven.containerapps.config.AppContainerMavenConfig;
+import com.microsoft.azure.maven.containerapps.config.DeploymentType;
+import com.azure.resourcemanager.appcontainers.models.EnvironmentVar;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Collections;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class AppContainerMavenConfigTest {
```

Comment:
The test quality in this file is not high. Do we really need to add tests for Getters / Setters?